justfile

```diff
@@ -37,6 +37,59 @@ build-cw-ics08-wasm-eth:
 build-relayer-image:
     docker build -t eureka-relayer:latest -f programs/relayer/Dockerfile .
 
+# Build the ibc-attestor docker image
```

Comment:
Not a huge fan of so many commands here, what is the core we actually need? And they should be more concise.